src/librustc_mir/transform/qualify_min_const_fn.rs

```diff
@@ -390,3 +386,30 @@ fn check_terminator(
         },
     }
 }
+
+/// Returns true if the `def_id` refers to an intrisic which we've whitelisted
+/// for being called from stable `const fn`s (`min_const_fn`).
+///
+/// Adding more intrinsics requires sign-off from @rust-lang/lang.
+fn is_intrinsic_whitelisted(tcx: TyCtxt<'a, 'tcx, 'tcx>, def_id: DefId) -> bool {
```

Comment:
I remember seeing this in a different PR, was that dropped in favour of including it in this one or is it just still pending and this will be redundant after it merges?

also is there any scope for using/deduplicating it with `intrisic_operation_unsafety` from src/librustc_typeck/check/intrinsic.rs since they seem to do the same things. or is it expected that the two will diverge?

Reply:
Yes these will diverge for e.g. `write_bytes` which writes to a raw pointer and thus can never be safe